specs/jsonschema-core.md

```diff
@@ -1002,12 +993,13 @@ applicators, applying the referenced schema to the instance.
 ##### Direct References with `$ref` {#ref}
 
 The `$ref` keyword is an applicator that is used to reference a statically
-identified schema. Its results are the results of the referenced schema.[^5]
+identified schema. Its results are the results of evaluating the referenced
+schema.[^5]
```

Comment:
This is less clear to me.  If you're going to require specificity that we're evaluating the referenced schema (which I think is sufficiently implied), then you need to require that same specificity of the `$ref` keyword results.

Personally I prefer this unchanged.

Reply:
That was how I originally wrote it, then changed it in the end. I do feel like just saying "the results" is less precise than it should be, but the revised sentence feels awkward. I'll revert the change.